### PR TITLE
Red text in terminal will be shown when find the target MAC addresses

### PR DESCRIPTION
**Pull request by syc728**

### CODE REVIEW

**Comment by syc728** [comment]
Please review the PR #37 @schollz 
Added an argument (--targetmacs) to read the file that contains the list of MAC addresses.
If find the MAC address in the list, output MAC address and rssi in the terminal with red.

**Comment by syc728** [comment]
Please review the PR #33 @schollz
Added an argument (--targetmacs) to read the file that contains the list of MAC addresses.
If find the MAC address in the list, output MAC address and rssi in the terminal with red.

**Comment by schollz** [comment]
Thanks @syc728!!  I will be able to take a look later this week!

File: howmanypeoplearearound/__main__.py [hunk 1]
```diff
@@ -11,6 +11,9 @@
 
 from howmanypeoplearearound.oui import oui
 from howmanypeoplearearound.analysis import analyze_file
+
+# import colors
+from colors import *
```
Comment (schollz) [inline howmanypeoplearearound/__main__.py]: `colors` -> `howmanypeoplearearound.colors`

**Comment by schollz** [comment]
Great, thanks!

**Pull request merged**